interfaces/builtin/fwupd.go

```diff
@@ -71,18 +84,49 @@ const fwupdPermanentSlotAppArmor = `
   /sys/firmware/efi/efivars/ r,
   /sys/firmware/efi/efivars/** rw,
 
+  /proc/modules r,
+  /run/udev/data/* r,
+  /sys/kernel/security/lockdown r,
+  owner @{PROC}/@{pid}/mountinfo r,
+  /dev/cpu/*/msr rw,
+
+  owner @{PROC}/@{pid}/mounts r,
+  /run/mount/utab r,
+  # Realtek MST plugin
+  /dev/i2c-[0-9]* rw,
+
+  /sys/kernel/security/tpm*/binary_bios_measurements r,
+  /dev/tpm* rw,
+  /dev/tpmrm* rw,
+  /sys/power/mem_sleep r,
+  /proc/swaps r,
+  /proc/sys/kernel/tainted r,
```

Comment:
nitpick, it feels like this fist part of additions could be a bit more organized with like grouped with like